### ISSUE TITLE
Installing as plugin on Netlify but no way to set sitemap filename

### ISSUE THREAD

**Issue by darkkilltec** [cdeleeuwe/netlify-plugin-submit-sitemap#35]
Hello,

I wanted to use this Plugin directly from Netlify. I am building my Site with Astrojs and have a package which generates 2 sitemap files. sitemap-index.xml and sitemap-0.xml.
I see that I could do this via file based, but is it not possible to configure this on Netlify via Plugin?

**Comment by cdeleeuwe** [comment]
@darkkilltec Yes, this is a limitation in Netlify and don't think the plugin can solve this.
